rules/bash.mk

```diff
@@ -1,14 +1,12 @@
 # bash package
 #
-# Created to patch memory-leak issue in the bash-package included in Debian-8 (Jessie)
-# release. This rule file, and the associated building-infra created to solve this
-# bug (src/bash/), should be eliminated once the migration to Debian-9 (Stretch) is
-# completed.
+# Created to patch plugin support in the bash-package included in Debian-10 (Buster)
+# release.
 
-# Bash major release-number corresponding to Debian-8 (Jessie)
-BASH_VERSION_MAJOR = 4.3
-# Bash complete release-number. This image contains all 4.3 fixes  up to patch '42'.
-BASH_VERSION_FULL = $(BASH_VERSION_MAJOR)-14
+# Bash major release-number corresponding to Debian-10 (Buster)
+BASH_VERSION_MAJOR = 5.0
```

Comment:
>5.0 [](http://example.com/codeflow?start=21&length=3)

Bullseye is using even newer version. It is not a good idea to build an old version for it.

Reply:
Bullseye using 5.1-2+b3 version: https://packages.debian.org/stable/shells/bash
will change to this version.